RecoTracker/MkFitCore/interface/Hit.h

```diff
@@ -192,8 +192,8 @@ namespace mkfit {
     struct PackedData {
       unsigned int detid_in_layer : 12;
       unsigned int charge_pcm : 8;  // MIMI see set/get funcs; applicable for phase-0/1
-      unsigned int span_rows : 3;
-      unsigned int span_cols : 3;
+      unsigned int span_rows : 5;
+      unsigned int span_cols : 5;
```

Comment:
to avoid magic numbers, perhaps use smth like 
https://github.com/trackreco/cmssw/blob/21244086c45e1318b3f8c0ec621258c76a33958a/RecoTracker/MkFitCore/interface/Track.h#L202-L203

Reply:
Done (for all bits in that struct).